apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/PromptResourceTest.java

```diff
@@ -85,6 +88,14 @@ class PromptResourceTest {
 
     private static final WireMockUtils.WireMockRuntime wireMock;
     public static final String[] IGNORED_FIELDS = {"versionCount", "latestVersion", "template"};
+    public static final String TEMPLATE = """
```

Comment:
Any reason to introduce this duplicate and not use `PromptVersionManufacturer.TEMPLATE`?